clicache/src/impl/PdxTypeRegistry.hpp

```diff
@@ -39,64 +39,64 @@ namespace Apache
 
       namespace Internal
       {
-        public ref class PdxTypeRegistry
+        private ref class PdxTypeRegistry
         {
         public:
 
           PdxTypeRegistry(Cache^ cache) :m_cache(cache) {}
-					//test hook;
-					  int testGetNumberOfPdxIds();
-
-					//test hook
-					  int testNumberOfPreservedData();
-
-            void AddPdxType(Int32 typeId, PdxType^ pdxType);
-
-            PdxType^ GetPdxType(Int32 typeId);
-
-            void AddLocalPdxType(String^ localType, PdxType^ pdxType);
-
-            PdxType^ GetLocalPdxType(String^ localType);
-
-            void SetMergedType(Int32 remoteTypeId, PdxType^ mergedType);
-
-            PdxType^ GetMergedType(Int32 remoteTypeId);
-
-            void SetPreserveData(IPdxSerializable^ obj, PdxRemotePreservedData^ preserveData);
-
-            PdxRemotePreservedData^ GetPreserveData(IPdxSerializable^ obj);      
-
-           void clear();
-
-            Int32 GetPDXIdForType(Type^ type, native::Pool* pool, PdxType^ nType, bool checkIfThere);
-
-            Int32 GetPDXIdForType(PdxType^ type, native::Pool*);
-
-            Int32 GetEnumValue(EnumInfo^ ei);
-
-            EnumInfo^ GetEnum(Int32 enumVal);
+          //test hook;
+          int testGetNumberOfPdxIds();
+
+          //test hook
```

Comment:
Out of scope maybe, but can we hide these test hooks?

Reply:
PdxTypeRegistry is internal